com.ultimaker.cura.yml

```diff
@@ -0,0 +1,313 @@
+    build-commands:
+      - openblas_options="DYNAMIC_ARCH=1 DYNAMIC_OLDER=1 NO_STATIC=1 TARGET=GENERIC"
+      - make PREFIX=/app $openblas_options
+      - make PREFIX=/app $openblas_options install
+    sources:
+    - type: archive
+      url: https://github.com/xianyi/OpenBLAS/archive/v0.3.13.tar.gz
+      sha256: 79197543b17cc314b7e43f7a33148c308b0807cd6381ee77f77e15acf3e6459e
+
+  - name: geos
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DGEOS_ENABLE_TESTS=OFF
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://download.osgeo.org/geos/geos-3.7.3.tar.bz2
+        sha256: 02035ae4e0ad711fa5a5556d7712530029edacac364b5b9c3ade0ded865fca7e
+
+
+  # Based on https://github.com/Ultimaker/cura-build-environment/blob/4.12/projects/base_requirements.txt
+  - python-modules/base-requirements.json
+
+  - name: protobuf
+    buildsystem: autotools
+    config-opts:
+      - --with-zlib=false
+      - --prefix=/app
+    sources:
+      - type: archive
+        url: https://github.com/protocolbuffers/protobuf/archive/refs/tags/v3.15.7.tar.gz
+        sha256: efdd6b932a2c0a88a90c4c80f88e4b2e1bf031e7514dbb5a5db5d0bf4f295504
+
+  - name: sip
+    buildsystem: simple
+    build-commands:
+      - python3 configure.py --bindir=/app/bin --destdir=/app/lib/python3.9/site-packages --incdir=/app/include --pyidir=/app/lib/python3.9/site-packages --sipdir=/app/share/sip
+      - make -j`nproc`
+      - make install
+    sources:
+      - type: archive
+        url: https://www.riverbankcomputing.com/static/Downloads/sip/4.19.25/sip-4.19.25.tar.gz
+        sha256: b39d93e937647807bac23579edbff25fe46d16213f708370072574ab1f1b4211
+
+  - name: pyqt
+    buildsystem: simple
+    build-commands:
+      - python3 configure.py --no-designer-plugin --no-tools --no-qml-plugin --sysroot /app --sip /app/bin/sip --sip-incdir /app/include --confirm-license --assume-shared --disable QtSensors --disable QtSerialPort --disable QtMultimedia --disable QtWebChannel --disable QtLocation -verbose QMAKE_CFLAGS_RELEASE='-I/usr/include/python3.9' QMAKE_CXXFLAGS_RELEASE='-I/usr/include/python3.9'
+      - make -j`nproc`
+      - make install DESTDIR=/app
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/28/6c/640e3f5c734c296a7193079a86842a789edb7988dca39eab44579088a1d1/PyQt5-5.15.2.tar.gz
+        sha256: 372b08dc9321d1201e4690182697c5e7ffb2e0770e6b4a45519025134b12e4fc
+
+  - name: nlopt
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: git
+        url: https://github.com/stevengj/nlopt.git
+        commit: 41967f1981d82d8495c0b27151a126bc35ae0d96
+        tag: v2.6.2
+
+
+  - name: boostheaders
+    buildsystem: simple
+    build-commands:
+      - cp -r boost /app/include/boost
+    sources:
```

Comment:
```suggestion
    cleanup:
      - /include
    sources:
```

Reply:
ah yeah right. I tend to add the to the modules, but the top-level should be good enough.